src/NetworkPortMetrics.php

```diff
@@ -170,12 +170,14 @@ public function showMetrics(NetworkPort $netport)
             'label' => __('Input/Output megabytes'),
             'icon'  => $params['icon'],
             'color' => '#ffffff',
-            'distributed' => false
+            'distributed' => false,
+            'show_points' => false,
+            'line_width'  => 2,
         ];
 
        //display bytes graph
         echo "<div class='netports_metrics bytes'>";
-        echo Widget::multipleLines($bytes_bar_conf);
+        echo Widget::multipleAreas($bytes_bar_conf);
```

Comment:
Should this be changed in 10.0/bugfixes too ?

Reply:
not necessary, this enhance the view, but i'm not sure it will be ok with chartist